spring-cloud-gcp-autoconfigure/src/test/java/org/springframework/cloud/gcp/autoconfigure/datastore/GcpDatastoreAutoConfigurationTests.java

```diff
@@ -35,16 +42,27 @@
  * Tests for Datastore auto-config.
  *
  * @author Chengyuan Zhao
+ * @author Raghavan N S
+ * @author Srinivasa Meenavalli
  */
 public class GcpDatastoreAutoConfigurationTests {
 
+	private DatastoreHealthIndicator datastoreHealthIndicator;
 	private ApplicationContextRunner contextRunner = new ApplicationContextRunner()
 			.withConfiguration(AutoConfigurations.of(GcpDatastoreAutoConfiguration.class,
-					GcpContextAutoConfiguration.class, DatastoreTransactionManagerAutoConfiguration.class,
-					DatastoreRepositoriesAutoConfiguration.class))
+					GcpContextAutoConfiguration.class,
+					DatastoreTransactionManagerAutoConfiguration.class,
+					DatastoreRepositoriesAutoConfiguration.class,
+					DatastoreHealthIndicatorAutoConfiguration.class,
+					DatastoreHealthIndicatorConfiguration.class,
+					DatastoreHealthIndicator.class))
 			.withUserConfiguration(TestConfiguration.class)
 			.withPropertyValues("spring.cloud.gcp.datastore.project-id=test-project",
 					"spring.cloud.gcp.datastore.namespace-id=testNamespace");
+	@Before
+	public void setUp() {
+		datastoreHealthIndicator = new DatastoreHealthIndicator(mock(Datastore.class));
```

Comment:
What is the purpose of this health indicator instance? Doesn't the autoconfig class use its own `createHealthIndicator` method to create the indicators?